rholang/src/main/scala/coop/rchain/rholang/interpreter/Runtime.scala

```diff
@@ -67,7 +67,7 @@ object Runtime {
     F[
       Par,
       BindPattern,
-      OutOfPhlogistonsError.type,
+      InterpreterError,
```

Comment:
Kind of unrelated but do we still need the `CPARK` type? IntelliJ shows zero usages.

Reply:
Good point, killed it :)